lib/cucumber/core/gherkin/document.rb

```diff
@@ -3,11 +3,12 @@ module Cucumber
   module Core
     module Gherkin
       class Document
-        attr_reader :uri, :body
+        attr_reader :uri, :body, :language
 
-        def initialize(uri, body)
-          @uri = uri
-          @body = body
+        def initialize(uri, body, language=nil)
```

Comment:
Eugh who's doing that? How about we get rid of the default value (and the anglophile bias in the code) altogether, make it a mandatory parameter and have the caller pass `en` explicitly? Waddayareckon?